cmd/check.go

```diff
@@ -300,18 +300,27 @@ func check() {
 		outputs = append(outputs, op)
 	}
 
-	// Loop over enabled packages and evaluate.
-	enabledPackages := viper.GetStringSlice("enabled-packages")
+	type EnabledPackage struct {
+		Name          string
+		EnabledRules  []string `mapstructure:"enabled-rules"`
+		DisabledRules []string `mapstructure:"disabled-rules"`
+	}
```

Comment:
This breaks backward compatibility with the existing configuration files, isn't it?

If that is the case, I'd rather keep `enabled-packages` as it is, and have a separate section in the configuration for the whitelist and the blacklist.

Reply:
Something like:

```
enabled-packages:
  - "jetstack.io/pods"
  - "jetstack.io/services"
rule-selectors:
  - package: "jetstack.io/pods"
    enabled-rules:
      - rule_1
  - package: "jetstack.io/services"
   disabled-rules:
      - rule_2
```

Reply:
Good point, that won't work with old config files. Based on previous discussion with @charlieegan3 I would prefer to keep the enabled/disabled rules under the `enabled-packages` list as it's tidier. I will look into how that could be made backwards compatible.

Reply:
I've changed this so if it can't unmarshal into the struct it tries to read it as a string slice, like before, and logs a warning about it being deprecated.

Reply:
I'd favour simplicity over tidiness, but I am ok with that.